torch/csrc/api/include/torch/nn/modules/any.h

```diff
@@ -431,7 +431,11 @@ AnyModule::AnyModule(std::shared_ptr<ModuleType> module)
       torch::detail::has_forward<ModuleType>::value,
       "Can only store module with a forward() method that has a non-templatized"
       "return type into AnyModule (e.g. we cannot store nn::Sequential"
-      "into AnyModule, because its forward() method's return type is templatized)");
+      "into AnyModule, because its forward() method's return type is templatized."
```

Comment:
I think the same here: `... because its forward() method's argument type and return type are templatized`